buildSrc/src/main/kotlin/trikita/anvilgen/utils.kt

```diff
@@ -0,0 +1,5 @@
+package trikita.anvilgen
```

Comment:
Maybe we should drop "trikita" org mention at all?

Reply:
I was waiting for feedback on reverse domain of `dev.inkremental.anvil` or others for the project before switching packages on code. As for generated part, we should have it configurable. But for latter to work some kind of metadata is needed to come with generated code so that dependencies are processed in tandem, not entirely separately like now.

Reply:
Let's leave it for separate PR, this one already gets unrelated changes.